CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # zhinst-qcodes Changelog
 
+## Version 0.5.4
+* Fix issue that prevented connecting to different dataserver on the same host.
```

Comment:
Fixed issue that prevented connecting to different data servers on the same host.